datafusion/substrait/src/logical_plan/consumer.rs

```diff
@@ -880,6 +886,42 @@ pub async fn from_substrait_rex(
                 ScalarFunctionType::ILike => {
                     make_datafusion_like(true, f, input_schema, extensions).await
                 }
+                ScalarFunctionType::IsNull => {
+                    let arg = f.arguments.first().ok_or_else(|| {
```

Comment:
I don't know if it matters, but this code doesn't check for `f.arguments.len() > 1` so I think it will silently ignore any arguments after the first.

The same comment applies to `IsNotNull`

Reply:
On review, this is the same pattern used elsewhere in this PR

Reply:
Thanks for merging my PR! I think I could add checks for arg length here and also in other places where they are required in another PR

Reply:
Thank you @tgujar  -- A follow on to make the argument checking handle too many arguments would be most appreciated. Thank you 🙏